pom.xml

```diff
@@ -199,26 +192,49 @@
                                 <fileSet>
                                     <directory>${project.build.directory}/</directory>
                                     <includes>
-                                        <include>jacoco-it.exec</include>
-                                        <include>jacoco-udf.exec</include>
+                                        <include>jacoco*.exec</include>
                                     </includes>
                                 </fileSet>
                             </fileSets>
-                            <destFile>${project.build.directory}/aggregate-it.exec</destFile>
+                            <destFile>${project.build.directory}/aggregate.exec</destFile>
                         </configuration>
                     </execution>
                     <execution>
-                        <id>report-integration-with-udf</id>
+                        <id>report</id>
                         <phase>verify</phase>
                         <goals>
-                            <goal>report-integration</goal>
+                            <goal>report</goal>
                         </goals>
                         <configuration>
-                            <dataFile>${project.build.directory}/aggregate-it.exec</dataFile>
+                            <dataFile>${project.build.directory}/aggregate.exec</dataFile>
                         </configuration>
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.exasol</groupId>
+                <artifactId>project-keeper-maven-plugin</artifactId>
+                <version>0.10.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <modules>
```

Comment:
Does it have a module for maven release?

Reply:
yes